### ISSUE TITLE
Cancellation Tropical Cyclone Sigmet cannot be produced.

### ISSUE THREAD

**Issue by Mattonius5** [wmo-im/iwxxm#338]
### Bug description
IWXXM 2023-1: Using following TAC to use for conversion:
```html
WCPA07 PHFO 180843
KZAK SIGMET T16 VALID 180821/181321 PHFO-
KZAK OAKLAND OCEANIC FIR CNL SIGMET T15 180734/181321=
``` 
When you convert it, We generated following IWXXM.
[LYPA07_PHFO_180843.txt](https://github.com/user-attachments/files/17364986/LYPA07_PHFO_180843.txt)

This file fails Schematron validation:
"TropicalCycloneSIGMET.TropicalCycloneSIGMET-3: There shall only be one iwxxm:analysisCollection in iwxxm:TropicalCycloneSIGMET"

Since this is cancellation report, analysisCollection should not exist. If you decided to pass this rule by adding single analysisCollection, you get quite expectedly a different Schematron rule violation:
"SIGMET.SIGMET-3: A 'CANCELLATION' report should have appropriately filled elements including iwxxm:issueTime, iwxxm:issuingAirTrafficServicesUnit, iwxxm:originatingMeteorologicalWatchOffice, iwxxm:issuingAirTrafficServicesRegion, iwxxm:sequenceNumber, iwxxm:validPeriod, iwxxm:cancelledReportSequenceNumber, iwxxm:cancelledReportValidPeriod.
**Elements iwxxm:phenomenon and iwxxm:analysisCollection shall be absent**"
[LYPA07_PHFO_180843 - Edited.txt](https://github.com/user-attachments/files/17365109/LYPA07_PHFO_180843.-.Edited.txt).

### Questions, proposals:

1. Why was the rule "TropicalCycloneSIGMET-3" added? To enforce multiple CBs being reported under "iwxxm:supplementaryAnalysisCollection"?
2. I recommend enhancing the "TropicalCycloneSIGMET-3" not to trigger for Cancellation reports.

Kind Regards,
Matous

**Comment by blchoy** [comment]
True.  TropicalCycloneSIGMET.TropicalCycloneSIGMET-3 should be updated to only execute when TranslationFailedTAC is set, permissibleUsage="NON-OPERATIONAL" or isCancelReport is set.
